rskj-core/src/main/java/org/ethereum/core/PendingTransaction.java

```diff
@@ -64,7 +64,7 @@ public long getBlockNumber() {
 
     public byte[] getSender() {
```

Comment:
I think this method is not used anywhere, in that case the `byte[] sender` field can be removed

Reply:
Just removed the whole class!